CASMCMS-7975: Add more information on CFS inventory and limit (#1368)
Cray-HPE/docs-csm: operations/configuration_management/Specifying_Hosts_and_Groups.md

# Specifying Hosts and Groups

> When using CFS, there are many steps where you may need to specify the hosts CFS should configure.  This can be done by specifying individual hosts, or groups of hosts.  However there are several places where a user may have to provide this information, particularly groups, and depending on where this information is provided, the behavior can change greatly.

Reviewer: ```suggestion
When using the Configuration Framework Service (CFS), there are many steps where users may need to specify the hosts that CFS should configure. This can be done by specifying individual hosts, or groups of hosts. There are several places where a user may need to provide this information, particularly groups, and depending on where this information is provided, the behavior can change greatly.
```

### Inventories

> CFS has multiple options for generating inventories, but regardless of which option is used, the information is then converted into an Ansible inventory/hosts file.  For more information on Ansible inventory, see the official [Ansible Inventory Documentation](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html).  The inventory is the list of components that Ansible can run against.  Anything not in the inventory is unknown to Ansible.  This is also the place where components can be placed into groups so that they can be easily referenced together.

Reviewer: ```suggestion
CFS has multiple options for generating inventories, but regardless of which option is used, the information is then converted into an Ansible inventory/hosts file. The inventory is the list of components that Ansible can run against.  Anything not in the inventory is unknown to Ansible. Components in an inventory can be placed into groups so that they can be easily referenced together when specified by CFS.

For more information on Ansible inventory, see the official [Ansible Inventory Documentation](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html).
```

# Specifying Hosts and Groups

When using the Configuration Framework Service (CFS), there are many steps where users may need to specify the hosts that CFS should configure. This can be done by specifying individual hosts, or groups of hosts. There are several places where a user may need to provide this information, particularly groups, and depending on where this information is provided, the behavior can change greatly.

## Inventories

> CFS has multiple options for generating inventories, but regardless of which option is used, the information is then converted into an Ansible inventory/hosts file. The inventory is the list of components that Ansible can run against.  Anything not in the inventory is unknown to Ansible. Components in an inventory can be placed into groups so that they can be easily referenced together when specified by CFS.

Reviewer 2: ```suggestion
CFS has multiple options for generating inventories, but regardless of which option is used, the information is then converted into an Ansible inventory/hosts file. The inventory is the list of components that Ansible can run against. Anything not in the inventory is unknown to Ansible. Components in an inventory can be placed into groups so that they can be easily referenced together when specified by CFS.
```
